src/Domain/Apprenticeship/Events/PriceChangeApprovedHandler.cs

```diff
@@ -37,4 +37,37 @@ public async Task Handle(PriceChangeApproved @event, CancellationToken cancellat
             await _messageSession.Publish(apprenticeshipCreatedEvent);
         }
     }
+
+    public class StartDateChangeApprovedHandler : IDomainEventHandler<StartDateChangeApproved>
```

Comment:
Any reason this isn't in its own file? or in the same file as the event?

Reply:
sorted